docs/chainlink-keepers/utility-contracts.md

```diff
@@ -0,0 +1,112 @@
+---
+layout: nodes.liquid
+section: smartContract
+date: Last Modified
+title: 'Keeper Utility Contracts'
+whatsnext:
+  {
+    'FAQs': '/docs/chainlink-keepers/faqs/',
+  }
+---
+
+# Overview <!-- omit in toc -->
+This guide explains the usages of Keeper utility contracts, specifically the [`EthBalanceMonitor` contract](https://github.com/smartcontractkit/upkeep-contracts/blob/master/contracts/upkeeps/EthBalanceMonitor.sol). This Keeper contract monitors and funds Ethereum addresses that developers may need to top up frequently based on a configurable threshold. As a result, nodes are funded automatically. 
+<p>
+</p>
+
+Once the contract is deployed, developers can go to [keepers.chain.link](https://keepers.chain.link/) to register Upkeep and run the contract. To take full advantage of the Keepers automation infrastructure, read all of the documentation to understand the features of Chainlink Keepers.
+<p>
+</p>
+
+**Table of Contents**
++ [`EthBalanceMonitor` Overview](#ethbalancemonitor-overview)
++ [Functions](#functions)
+  + [`setWatchList` Function](#setwatchlist-function)
+  + [`setKeeperRegistryAddress` Function](#setkeeperregistryaddress-function)
+  + [`setMinWaitPeriodSeconds` Function](#setminwaitperiodseconds-function)
+  + [`topUp` Function](#topup-function)
+
+# `EthBalanceMonitor` Overview
+`EthBalanceMonitor` is Ownable, Pausable, and Keeper-compatible. Each of these components are explained in more detail:
+
+- **Ownable**: The contract has an owner address, and provides basic authorization control functions. This simplifies the implementation of "user permissions" and allows for transer of ownership. 
+- **Pausable**: This feature allows the contract to implement an pause/unpause mechanism that can be triggered by the owner.
+- **Keeper-compatible**: The `KeeperCompatibleInterface` is necessary to create Keeper-compatible contracts. To learn more about the `KeeperCompatibleInterface` and its uses and functions, refer to [Making Keeper-compatible Contracts](../compatible-contracts/).
+
+> ⚠️ Note on Owner Settings
+> Aside from certain features listed below, only owners can withdraw funds and pause/unpause the contract. If the contract is paused/unpaused, it will affect `checkUpkeep`, `performUpkeep`, and `topUp` functions.
```

Comment:
```suggestion
> Aside from certain features listed below, only owners can withdraw funds and pause or unpause the contract. If the contract is paused or unpaused, it will affect `checkUpkeep`, `performUpkeep`, and `topUp` functions.
```